public/locales/en/app.ftl

```diff
@@ -0,0 +1,35 @@
+-product-name = Firefox Monitor
+
+layout-support = Support
+layout-feedback = Give Feedback
+layout-terms = Terms&nbsp;and&nbsp;Privacy
+layout-copyright = Portions of this content are &copy; 1998-2018 by individual mozilla.org contributors. <br />Content available under a <a {{> analytics/link_event eventLabel="Creative Commons"}} href="https://www.mozilla.org/foundation/licensing/website-content/" target="_blank" rel="noopener">Creative Commons license</a>.
```

Comment:
Not sure how well fluent supports nested HTML, but I'm pretty sure it isn't great from a localizer standpoint since it can get pretty complex (for example, localizers will have to ignore the handlebars partial and that `eventLabel` which shouldn't be localized). I think l20n had ways to avoid this, but i don't know about fluent.

Reply:
@flodolo ^ ?